share/exchanges_trading_days_aliases.yml

```diff
@@ -0,0 +1,29 @@
+---
+##
+# This is a bitmap of the day number mod 7: [0..6] => [Sunday .. Saturday] that represent whether the exchange is open or not on that day.
+# All markets are traded in either of these three group . Example: Volatility Indices trade on everyday, Forex and most of the indices trade on weekedays and Middle east indices trade on Sun to Thurs.
```

Comment:
there is no other mention of "Volatility Indices" in this module so it seems to have no place in this repo. Remember that this is an open-souce repo that has nothing to do with binary.com